### PR TITLE
update Mqtt Client

### PR DESCRIPTION
**Pull request by MichaelDvP**
Bert has fixed the issues in mqtt. But i leave the default mem level in the library and added a check in mqtt.c. Let's hope that this was the issue from #1264 
#1271 is included

### CODE REVIEW

File: src/web/WebStatusService.cpp [hunk 1]
```diff
@@ -166,9 +166,9 @@ void WebStatusService::webStatusService(AsyncWebServerRequest * request) {
     if (Mqtt::enabled()) {
         statJson       = statsJson.createNestedObject();
         statJson["id"] = 5;
-        statJson["s"]  = Mqtt::publish_count();
+        statJson["s"]  = Mqtt::publish_count() - Mqtt::publish_fails();
```
Comment (proddy) [inline src/web/WebStatusService.cpp:169]: nice!

**Review by proddy — APPROVED** [review]

**Comment by proddy** [comment]
really great work hunting down the root cause for this freeze/restarts Michael.

**Comment by proddy** [comment]
And @MichaelDvP I think we need to make a v3.6.1 soon, based on what we have in dev

File: src/web/WebStatusService.cpp (hunk 1 above)
Comment (MichaelDvP) [reply src/web/WebStatusService.cpp:169]: statJson["s"] is for success, publish_count() is all messages. Same correction for quality calculation.

**Comment by MichaelDvP** [comment]
> And @MichaelDvP I think we need to make a v3.6.1 soon, based on what we have in dev

Yes.

**Pull request merged**